.github/workflows/test.yml

```diff
@@ -71,19 +89,51 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.8']
-        django: ['3.2']
-        wagtail: ['2.13', '2.14', '2.15', '2.16', '3.0']
+        python: ['3.8', '3.9', '3.10']
+        django: ['3.2', '4.0']
+        wagtail: ['2.16', '3.0', '4.0']
         postgres: ['10.8']
         include:
-          - wagtail: '2.16'
-            django: '4.0'
+          - python: '3.7'
+            django: '3.2'
+            wagtail: '2.15'
             postgres: '10.8'
-            python: '3.8'
-          - wagtail: '3.0'
+          - python: '3.8'
+            django: '3.2'
+            wagtail: '2.15'
+            postgres: '10.8'
+          - python: '3.9'
+            django: '3.2'
+            wagtail: '2.15'
+            postgres: '10.8'
+          - python: '3.10'
+            django: '3.2'
+            wagtail: '2.15'
+            postgres: '10.8'
+          - python: '3.8'
+            django: '4.1'
+            wagtail: '4.0'
+            postgres: '10.8'
+          - python: '3.9'
+            django: '4.1'
+            wagtail: '4.0'
+            postgres: '10.8'
+          - python: '3.10'
+            django: '4.1'
+            wagtail: '4.0'
+            postgres: '10.8'
+          - python: '3.11'
```

Comment:
Started adding python 3.11 and wagtail 4.1 testing here for postgres, Also rearranged these variables.